libs/csm/rest/csm_rest_system_health.py

```diff
@@ -626,3 +626,54 @@ def cluster_operation_signal(
         self.log.info("%s operation on %s POST REST API response : %s",
                       operation, resource, response)
         return True, response
+
+    @RestTestLib.authenticate_and_login
+    @RestTestLib.rest_logout
+    def post_resource_signal(
+            self,
+            req_body: dict,
+            resource: str):
+        """
+        This method POST resource failure/shutdown event/signal to cluster
+        :param req_body: POST operation request body
+        :param resource: POST request body info
+        :return: bool, POST API response
+        """
+        # Building request url to POST resource failure signal
+        endpoint = "{}/{}".format(self.config["cluster_operation_endpoint"], resource)
+        headers = self.headers
+        conf_headers = self.config["Login_headers"]
+        headers.update(conf_headers)
+        self.log.info("POST REST API Endpoint :", endpoint)
+        # Fetching api response
+        response = self.restapi.rest_call("post", endpoint=endpoint, data=json.dumps(req_body),
+                                          headers=headers)
+        if response.status_code != HTTPStatus.OK:
+            self.log.error("POST REST API response : %s", response)
+            return False, response
+        self.log.info("POST REST API response : %s", response.json())
+        return True, response.json()
+
+    @RestTestLib.authenticate_and_login
+    @RestTestLib.rest_logout
```

Comment:
same as above.

Reply:
> same as above.

same as above. If the same function we are going to use for control then we need to change the logic

Reply:
Get resource status we need for control pod as well for sure.